src/Menu/Menu.js

```diff
@@ -148,9 +148,10 @@ class Menu extends Component {
      */
     valueLink: PropTypes.object,
     /**
-     * The width of the menu. If not specified, the menu's width
-     * will be set according to the widths of its children, using
-     * proper keyline increments (64px for desktop, 56px otherwise).
+     * The width of the menu in string ("200px", "50vw") or number (200, 300)
```

Comment:
@janzenz `"50%"` works just fine, I think you're just misunderstand it. The containing element has to have a width for `Menu` to be a percentage of, just as with any other element. This isn't anything specific to` Menu` or `Material-UI`. 

Providing two specific examples makes it appear that they are the only supported options, even with the 'e.g.', unless we also append 'etc.': `'(e.g. "200px", "50vw" etc.)'` (messy), or call out every supported option - (verbose).

This isn't consistent with how other props are documented.

`vw` is a slightly unusual choice for a menu width in most circumstances.

Thanks for the contribution, but I think this PR adds more confusion that it removes.